src/main/java/com/todolist/app/springmvcboottodolist/models/Status.java

```diff
@@ -24,6 +24,9 @@ public enum Status {
     @Override
     public String toString() {
         String str = name().toLowerCase().substring(0, 1).toUpperCase() + name().toLowerCase().substring(1);
+        if(str.equals("Inprogress")){
```

Comment:
Is it possible to have field in the enum assosiated with the value, e.g.
```
enum Status {
     String title;
     
     IN_PROGRESS("In progress)
...
}
```
E.g. https://www.baeldung.com/java-enum-values